digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/AssetFiller.scala

```diff
@@ -568,3 +568,37 @@ object NumericalLimitFiller {
       informationSource = asset.informationSource), changeSet)
   }
 }
+
+class OneWayAssetFiller extends AssetFiller {
```

Comment:
Create a separated file with this class if we want to keep the OneWayAssetFillerSpec.scala file if not just move the code inside  AssetFillerSpec.scala file